cypress/e2e/shared/tasks.test.ts

```diff
@@ -145,9 +143,10 @@ http.post(url: "https://foo.bar/baz", data: bytes(v: "body"))`
 
   describe('When tasks already exist', () => {
     beforeEach(() => {
+      const TaskName = '🦄ask'
       cy.get<Organization>('@org').then(({id}: Organization) => {
         cy.get<string>('@token').then(token => {
-          cy.createTask(token, id)
+          cy.createTask(token, id, TaskName)
```

Comment:
Added an explicit taskName in this test and added that to the function just for readability because I will refernece task name in the tests later on.